integration/integration_test.go

```diff
@@ -4855,6 +4855,53 @@ func (s *IntSuite) TestBPFSessionDifferentiation(c *check.C) {
 	c.Fatalf("Failed to find command events from two different sessions.")
 }
 
+// TestExecEvents tests if exec events were emitted with and without PTY allocated
+func (s *IntSuite) TestExecEvents(c *check.C) {
```

Comment:
As we are gradually migrating off of gocheck, all new tests should be written with the testing package directly. There's a migration guide on slab but I could not find it quickly.

Reply:
All I could find is an [RFD for testing guidelines](https://github.com/gravitational/teleport/blob/master/rfd/0001-testing-guidelines.md) that talks about migrating towards Go tests. It does seem like a lot of refactoring will need to be done to migrate the integration tests to use the testing package. Maybe we can discuss this at some point this week to see if we can come up with a plan to start refactoring.